packages/adapters/wagmi/src/client.ts

```diff
@@ -335,22 +356,36 @@ export class WagmiAdapter implements ChainAdapter {
               await SIWEController.verifyMessage({
                 message,
                 signature: s.s,
-                cacao: signedCacao
+                cacao: signedCacao,
+                clientId
               })
+              isSuccessful1CA = true
             } catch (error) {
+              isSuccessful1CA = false
+              SIWEController.setIs1ClickAuthenticating(false)
```

Comment:
how are these flags different?

Reply:
`isSuccessful1CA` is to know if the 1CA flow was successful and run the `onSignIn` logic if provided in the `siweConfig` and it's only used here.
`setIs1ClickAuthenticating` is used to show a loader on the QR code screen and it's in the global state